packages/data-access/src/data-access.ts

```diff
@@ -305,7 +305,8 @@ export default class DataAccess implements DataAccessTypes.IDataAccess {
     return transactionsAndMetaPerBlocks.reduce(
       (accumulator: DataAccessTypes.IReturnGetTransactions, elem) => ({
         meta: {
-          storageMeta: accumulator.meta.storageMeta.concat(elem.storageMeta),
+          // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
the initial value sets the storageMeta, so I feel it's better to disable eslint than changing the logic